config/nephe.yml

```diff
@@ -480,6 +495,24 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - crd.antrea.io
+  resources:
+  - externalnodes
+  verbs:
+  - create
+  - delete
+  - get
+  - list
```

Comment:
do you require list and patch permission. Wouldnt just create, delete and get sufficient?

Reply:
I tried without the list permission, it failed with an error saying unable to list the resources.